Admin.js

```diff
@@ -23,11 +23,8 @@ var Admin = {
         Admin.log("[core|shared_setup] Register services on", subject);
```

Comment:
- eslint: Strings must use doublequote.
- eslint: Missing space before value for key 'border-radius'.
- eslint: Strings must use doublequote.